src/js/components/addData/AddDataContent.jsx

```diff
@@ -51,25 +51,14 @@ export default class AddDataContent extends React.Component {
 
```

Comment:
While you're in this file anyway, I THINK some of the imports aren't used, double-check?